src/robot.coffee

```diff
@@ -269,6 +269,8 @@ class Robot
     user    = process.env.EXPRESS_USER
     pass    = process.env.EXPRESS_PASSWORD
     stat    = process.env.EXPRESS_STATIC
+    port    = process.env.EXPRESS_PORT or process.env.PORT or 8080
+    address = process.env.BIND_ADDRESS or '0.0.0.0'
```

Comment:
`BIND_ADDRESS` was added in https://github.com/github/hubot/pull/609 . I'd be fine namespacing it, as long as the old BIND_ADDRESS is used as well.